.changeset/markdown-alternate-link.md

```
@@ -0,0 +1,11 @@
+---
+'mastra-docs': patch
+---
+
+Added a markdown alternate link to every docs page. Each page now names its own markdown version in the HTML head:
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)